docs/platform/integrations/metrics/prometheus.rst

```diff
@@ -0,0 +1,159 @@
+       :scale: 55%
+
+   |
+   | Remember to click the ``+`` button after naming your endpoint
+     otherwise it will not be saved when you leave the page. You will
+     know it is set up correctly when an automatically generated
+     username, password, and port all appear in the UI.
+   | Usually you only need one of these per project (as it can be used
+     for all services in the same project) so if you can see one has
+     already been created then you can skip this step.
+
+#. | Navigate to your service that you
+     would like to monitor and click ``Manage integrations``
+
+   .. image:: /images/platform/integrations/prometheus-manage-integrations.png
+
+#. | Choose the ``Prometheus`` service integration by clicking
+     ``Use integration``
+
+   .. image:: /images/platform/integrations/prometheus-integration-select.png
+
+#. | Confirm you have selected the Prometheus endpoint configured
+     earlier in Step 2 and click ``Enable``
+
+   .. image:: /images/platform/integrations/prometheus-endpoint-select.png
+
+#. | Check the service overview page to see that the integration has
+     been successfully enabled and is now ``Active``
+
+   .. image:: /images/platform/integrations/prometheus-configured-integration.png
+
+#. | Copy the Prometheus connection information found from the top of
+     the service overview page under the new ``Prometheus`` tab
+
+   .. image:: /images/platform/integrations/prometheus-service-info.png
+
+
+
+After finishing these steps, the system will start an HTTP server on all
+nodes of the service that provide access to the metrics. Note that there
+can be roughly one minute delay until the metrics are available.
+
+Often the users have VPC enabled in their projects. If this was the
+case, the property **public_access.prometheus** needs to be enabled in
+the Advanced Configurations of the service. In this way it becomes
+possible to access metrics by using the public hostname.
+
+Configuring your Prometheus server
+-----------------------------
+
+To make Prometheus fetch metrics from Aiven servers you'll need to add a
+new scrape configuration with appropriate basic auth parameters (as seen on the
+Service Integrations page) and identify the servers to pull data from.
+
+Handling DNS resolution to multiple nodes
+'''''''''''''''''''''''''''''''''''''''''
+
+For any services that consist of multiple nodes and each node doesn't
+have its own unique DNS name, you need to use the ``dns_sd_configs``
+option for defining the servers with DNS type set to ``A`` . This causes
+Prometheus to resolve all the IP addresses associated with the DNS name
+and query all of those IP addresses directly. A side effect of using
+this IP resolution is that Prometheus expects the TLS certificate to be
+bound to the IP address of the hosts, not to the DNS name, so to make
+the connection work you must enable the ``insecure_skip_verify``
+setting.
+
+::
```

Comment:
Can we introduce what this code sample actually is or does? I think it's a section of the prometheus configuration file, but let's make it clear what it is and where it goes.

Reply:
Yes you are right, its a section of prometheus configuration.  This is the description of the configuration but I think the placement doesn't make it obvious. "To make Prometheus fetch metrics from Aiven servers you’ll need to add a new scrape configuration with appropriate basic auth parameters (as seen on the Service Integrations page) and identify the servers to pull data from."